### PR TITLE
Adding glenny.hackclub.com

### PR DESCRIPTION
**Pull request by BobbedBob**
# Adding glenny.hack.club

## Description

I am adding glenny.hack.club which points to my Github Pages site currently at ghc.bobbedbob.io.
This is for Glenny Hack Club which is an official Hack Club that will run in 2025 at Glen Waverley Secondary College, Melboburne, Australia.

### CODE REVIEW

**Comment by saumilthecode** [comment]
i dont really maintain this but i think you updated the wrong one

hack.club is for shortlinks.
the one you want to put your club website on is the hackclub.com file > https://github.com/hackclub/dns/blob/main/hackclub.com.yaml

**Comment by BobbedBob** [comment]
> i dont really maintain this but i think you updated the wrong one
> 
> hack.club is for shortlinks. the one you want to put your club website on is the hackclub.com file > https://github.com/hackclub/dns/blob/main/hackclub.com.yaml

Is there some sort of evidence to this reply? I assumed that when it says:
> ...under domains following the format hackclub.xxx(or other domains referring to hackclub), are only available for clubs...
That I can choose which kind of domain I would like to use for my club from the existing .yaml files.

File: hack.club.yaml [hunk 1]
```diff
@@ -8,6 +8,9 @@ ahoy:
 hack.club:
   - type: ALIAS
     value: cname.vercel-dns.com.
+glenny:
+  - type: CNAME
+    value: bobbedbob.github.io
```
Comment (Muirrum) [inline hack.club.yaml:13]: Please move to hackclub.com. hack.club is reserved for internal use.

Comment (Muirrum) [inline hack.club.yaml:13]: Please move to hackclub.com, hack.club is reserved for internal use.

**Review by Muirrum — CHANGES_REQUESTED** [review]

**Review by alx-alexpark — COMMENTED** [review]
Using gh pages generally requires a verification record, does github ask you to add one?

**Comment by BobbedBob** [comment]
> Using gh pages generally requires a verification record, does github ask you to add one?

Nope, not at all.

**Review by alx-alexpark — APPROVED** [review]
lgtm

**Pull request merged**